helm-chart/renku/requirements.yaml

```diff
@@ -1,8 +1,11 @@
 dependencies:
 - name: renku-core
   alias: core
-  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 2.3.2
+  repository: "file:///home/zenon/DEV/ETH/renku-python/helm-chart/renku-core"
+  version: 2.4.0
+  import-values:
+  - child: versions
+    parent: global.core.versions
```

Comment:
So the only reason for this is to then use `global.core.versions` in the renku-gateway helm chart?